react/store/createReturnRequest/components/ConfirmAndSubmit.tsx

```diff
@@ -28,8 +29,10 @@ export const ConfirmAndSubmit = ({ onPageChange, items }: Props) => {
     MutationCreateReturnRequestArgs
   >(CREATE_RETURN_REQUEST)
 
-  // temp state just to show request id on UI
-  const [requestId, setRequestId] = useState('')
+  const { navigate } = useRuntime()
+
+  const [showSubmittedStatus, setShowSubmittedStatus] = useState(false)
+  const [confirmationStatus, setConfirmationStatus] = useState('success')
```

Comment:
We should not initiate a state with a future value (success will only be the case after submitting the useMutation). Also, when we are controlling state with use mutation and we know all possible values for it, we should type the `useState` function.